CONTRIBUTING.md

```diff
@@ -35,7 +35,14 @@ so that we adhere to [Semantic Versioning v2.0.0](http://semver.org/spec/v2.0.0.
 6. Make sure you [ran the tests](#running-tests) and didn't break anything. That will save some time on
 [GitHub actions](https://github.com/Behat/Behat/actions)
 7. Commit your code and submit a Pull Request, providing a clear description of the change, including
-the motivation for the proposal
```

Comment:
can this whole section be moved to a PR template? then it will not be missed

Reply:
Absolutely, that's on the list.